traffic_ops/traffic_ops_golang/api/api.go

```diff
@@ -994,3 +994,11 @@ func CreateDeprecationAlerts(alternative *string) tc.Alerts {
 		return tc.CreateAlerts(tc.WarnLevel, "This endpoint is deprecated, and will be removed in the future")
 	}
 }
+
+// Sorting alphabetically for a given readertype (eg: TOCDN, TODeliveryService, TOOrigin etc).
+func Sort(readerType *APIInfo, param string) {
```

Comment:
Should we call this `DefaultSort`? Also, the godoc comment should start with the function name.

Reply:
Sure, I am ok with changing the name of the function.